CMakeLists.txt

```diff
@@ -42,3 +42,6 @@ target_include_directories(facil.io
   PUBLIC  lib/facil/redis
 )
 
+set_property(TARGET facilio_objlib PROPERTY POSITION_INDEPENDENT_CODE 1)
```

Comment:
Interesting... I'm no CMake user / expert - what do these lines do?

Does this replace the option of using facil.io as a source code library?

Reply:
It basically adds a `CFLAG` named `-fPIC` which tells the compiler to generate position independent code.
For shared libraries this is a must as the start address of the memory mapped library can be different to the default image base address set by the ELF linker.